.changes/fix-core-ios-async-panic.md

```diff
@@ -0,0 +1,5 @@
+---
+tauri: "patch:bug"
+---
+
+Fixed an issue that caused iOS apps to panic with an async `run()` entrypoint.
```

Comment:
```suggestion
Fixed an issue that caused iOS apps to panic when using an async function for `tauri::mobile_entry_point`.
```

Reply:
I abused my powers in this repo once (to hotfix the cli, just squeezed this one in) and then this x) I'll fix it in the covector pr.